### ISSUE TITLE
unable to install on python39

### ISSUE THREAD

**Issue by thomaswillecomme** [genicam/harvesters#213]
**Describe the bug**
unable to install on up to date Arch linux system with python 3.9 due to strict dependency to genicam

**To Reproduce**
$ pip install harvesters --no-cache-dir
Defaulting to user installation because normal site-packages is not writeable
Collecting harvesters
  Downloading harvesters-1.2.8-py3-none-any.whl (93 kB)
     |████████████████████████████████| 93 kB 1.2 MB/s 
Collecting numpy
  Downloading numpy-1.19.4-cp39-cp39-manylinux2010_x86_64.whl (14.5 MB)
     |████████████████████████████████| 14.5 MB 817 kB/s 
ERROR: Could not find a version that satisfies the requirement genicam==1.0.0 (from harvesters->-r requirements.txt (line 1)) (from versions: none)
ERROR: No matching distribution found for genicam==1.0.0 (from harvesters->-r requirements.txt (line 1))

**Expected behavior**
pull dependency genicam~1.0, not genicam==1.0.0

**Screenshots**
$ pip search genicam
genicam (**1.0.1**)           - The official Python Binding for the GenICam GenApi & the GenTL Producers
camazing (0.9.0)          - Machine vision library for GenICam-compliant cameras
genicam2 (0.0.0.dev5472)  - The official Python Binding for the GenICam GenApi & the GenTL Producers
harvesters-util (0.1.0)   - Image acquisition & visualization library with Python (Maintained by the official GenICam committee)

**Desktop:**
 - OS: Arch Linux amd64
 - Python: 3.9

**Comment by kazunarikudo** [comment]
Hi, the `genicam` package has not supported Python 3.9 and it will be supported next year. Sorry for the inconvenience. Note that we as the GenICam committee would release `genicam` package version 1.1.0 but it does not support Python 3.9 either. That's just to bind the latest version of GenICam GenApi reference implementation, version 3.3.0. /Kazunari.

**Comment by kazunarikudo** [comment]
Update: Version 3.9 should be supported by the Fall-Winter 2021 release. Regards, Kazunari.

**Comment by Scarjit** [comment]
Hi,

is there any progress on this Issue ?

Python 3.6 security support ends on 23.12.2021
Recent NumPy versions (>= 1.20.0) require at least python > 3.6

**Comment by kazunarikudo** [comment]
@Scarjit Hi, thank you for your comment. We as the GenICam committee is preparing the GenICam reference implementation version 3.4 now and it will eventually release the Python package that supports Python 3.9. Thank you for your understanding. Regards, Kazunari.

**Comment by Gornoka** [comment]
Hi @kazunarikudo,
 is there any development on this topic ?

**Comment by kazunarikudo** [comment]
Yes, it’s been under voting process in the GenICam committee. We’ll have an official meeting in April/May so the package should be released a bit earlier than that. The next version will work with 3.10, too.

**Comment by kazunarikudo** [comment]
Hi All, the GenICam GenApi reference implementation version 3.4 has been officially approved. Once the binary appears on the EMVA website then I will upload the Python package to PyPI; technically speaking I can do that anytime but this is just a matter of the protocol. Thanks again for your patience. I highly appreciate that.

**Comment by Gornoka** [comment]
It appears as that the 3.4 Reference Application has been uploaded on the EMVA website, do you mind updating the package? 
sorry for being so pushy but I would really like to get rid of Python 3.8 in my project.

**Comment by fuzzdoug** [comment]
I noticed the GeniCam reference package 2022.06 doesn't come with "PythonWheels" for Win32_i86 versions.
If that is the case, is there a way to install GeniCam on python39 32bit?

**Comment by burnpanck** [comment]
@fuzzdoug: The python wheels are not included in the reference package, they are [uploaded to PyPI](https://pypi.org/project/genicam/#files). You have to wait until the GenICam comittee decides to upload a compatible wheel. That happened the last time ~15 months ago. For unexplainable reasons, they have decided only to do such a release after they have done a release of the GenICam package itself, but even then the pace is not clear.

Honestly, I don't understand why the release of the python wheels are coupled to the GenICam releases. After all, the reference implementation is supplied as a shared library, so no recompilation of the library is needed to build a new wheel for a different python version. If those wrappers were released open-source, then the community could compile wheels for all architectures for which a GenICam runtime is available.

I know, I'm ranting here, but the python support by the committee is extremely appalling: 3.8 is halfway through it's end-of-life duration now, 3.10 was released 9 months ago, and 3.9 was already released by the time they finally released wheels for 3.8. This wrapper can't be more complicated than scipy, so it should be possible to support the latest python version in the same timeframe scipy does (hint: scipy provided wheels for python 3.9 in nov 2020; four weeks after python 3.9 was released).
